ci-operator/step-registry/oadp/rosa/aws/sts/oadp-rosa-aws-sts-workflow.yaml

```diff
@@ -0,0 +1,22 @@
+workflow:
+  as: oadp-rosa-aws-sts
+  steps:
+    env:
+      COMPUTE_MACHINE_TYPE: "m5.2xlarge"
```

Comment:
Just curious, are the environment variable overrides here required by OADP or is this something specific to ROSA? If it is for OADP, should we consider making the changes in the oadp-ipi-aws workflow as well?